src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int START_VALUE = 0;
+    private static final String COMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeInfoToFile(calculateSupplyValue(readFromFile(fromFileName)),
+                calculateBuyValue(readFromFile(fromFileName)), toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                builder.append(line).append(COMA);
+                line = bufferedReader.readLine();
+            }
+            return builder.toString().split(COMA);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + fromFileName + e.getMessage());
+        }
+    }
+
+    private int calculateSupplyValue(String[] dataLines) {
```

Comment:
It's better to create one method - calculateReport and pass it to writeIntoFile method
and return String as 
```
SUPPLY + "," + supply + System.lineSeparator()
                    + BUY + "," + buy + System.lineSeparator()
                    + RESULT + "," + (supply - buy)
```

but use StringBuilder to avoid creating new String every time during concatenation